packages/jest-config/src/__tests__/normalize.test.js

```diff
@@ -1579,6 +1581,29 @@ describe('displayName', () => {
       }).toThrowErrorMatchingSnapshot();
     },
   );
+
+  it.each`
+    runner                  | color
```

Comment:
Seems like we are not using the color column in the test.

Reply:
You're right, ended up going for snapshots since we don't actually care about specific colors, just that they're reasonable and different ones